test/transportSpec.js

```diff
@@ -140,6 +140,13 @@ describe('Transport', function () {
       assert.equal(details.headers['header'], 'value')
     })
 
+    it('should throw an error if header value contains a line feed', function () {
+      this.params.headers = { 'Some-Header': 'value\nX-Header-Injection: Some-Value' }
```

Comment:
Add a test `this.params.headers = { 'Some-Header\nX-Header-Injection: Some-Value': 'value' }`